eternalcombat-api/src/main/java/com/eternalcode/combat/command/CombatTagImmunityCommand.java

```diff
@@ -1,35 +1,40 @@
-package com.eternalcode.combat.fight.tagout;
+package com.eternalcode.combat.command;
 
 import com.eternalcode.combat.config.implementation.PluginConfig;
+import com.eternalcode.combat.fight.tagout.FightTagOutService;
 import com.eternalcode.combat.notification.NotificationAnnouncer;
 import com.eternalcode.combat.util.DurationUtil;
-import dev.rollczi.litecommands.argument.Arg;
-import dev.rollczi.litecommands.command.execute.Execute;
-import dev.rollczi.litecommands.command.permission.Permission;
-import dev.rollczi.litecommands.command.route.Route;
-import org.bukkit.entity.Player;
-import panda.utilities.text.Formatter;
-
+import dev.rollczi.litecommands.annotations.argument.Arg;
+import dev.rollczi.litecommands.annotations.command.Command;
+import dev.rollczi.litecommands.annotations.context.Context;
+import dev.rollczi.litecommands.annotations.execute.Execute;
+import dev.rollczi.litecommands.annotations.permission.Permission;
 import java.time.Duration;
 import java.time.Instant;
 import java.util.UUID;
+import org.bukkit.entity.Player;
+import panda.utilities.text.Formatter;
 
-@Route(name = "tagout", aliases = "tagimmunity")
-public class TagOutCommand {
+@Command(name = "tagout", aliases = "tagimmunity")
+public class CombatTagImmunityCommand {
```

Comment:
changing the class name and package does not fit into the rest of the "tagout" code